trunner/test_runner.py

```diff
@@ -20,6 +24,69 @@ def resolve_project_path():
     return project_dir
 
 
+def init_logdir():
+    """Inits log directories if it's needed."""
+    if not trunner.ctx.logdir:
+        return
+
+    logdir = trunner.ctx.logdir
+    internal_logdir_names = ("out", "in", "inout")
```

Comment:
I'm wondering if this is a good directory structure for logs. I think it would make more sense to group these logs by test, with directories named after each tests, and files inside named log.out, log.in and log.inout. Alternatively, we could avoid creating additional directories and simply save the logs as name.in, name.out, name.inout.

With this approach, when a test fails, we can simply go into the directory named after that test, or look for name.*, rather than having to navigate into the in, out and inout directories. This should also simplify the code.

What do you think?

Reply:
Yeah, we discussed it and your solution seems to be more suitable. It will be available in the next force push.